lib/good_job/configuration.rb

```diff
@@ -51,5 +51,13 @@ def poll_interval
         1
       ).to_i
     end
+
+    def before_seconds_ago
```

Comment:
I think this method should be named "cleanup_preserved_jobs_before_seconds_ago". It's a little unsymmetrical because the command line option isn't the whole phrase, but the longer phrase seems like the appropriate message to ask of the configuration object.

Reply:
Yes, that came to my mind at first, but decided not to do as I wasn't going to change the public configuration option GoodJob asks from user.

Changes now